### PR TITLE
Responsiveness Fixed

### LINKED ISSUE #101: Improve website responsiveness

### PR DESCRIPTION
**Pull request by aliraza944**
- Resolve #101 

- [x] About page responsiveness resolved
- [x]  Made the iframe to be adaptive on all screens. You can change the height of the iframe to adjust to your suitability.
- [x] Explore data flex layout set the column for small screens and row for large screens.
- [x]  Heights adjusted so that the footer on the about page stays on the bottom

### CODE REVIEW

File: src/views/Home.js [hunk 1]
```diff
@@ -23,8 +23,8 @@ export function DefaultPage({ children }) {
   });
 
   return (
-    <div className="h-screen w-100">
-      <div className="h-full w-full absolute z-10 flex items-center">
+    <div className=" w-full">
```
Comment (d-rita) [inline src/views/Home.js:26]: @aliraza944 can we retain the height and width specifications, i.e. "h-screen w-100" in order to be able to display the map? With just "w-full" and no inherited width property to define it, nothing is set and the map does not show.

**Review by d-rita — CHANGES_REQUESTED** [review]
Tested the PR and the changes are good. The Reports page and the About page are responsive to different screen sizes with no UI elements overlapping each other. 
1. Explore Reports Page:
explore reports page - tested
2. About Page: 
about page - tested

However, the Home page does not have the map displayed with the proposed change in the width and height specs. Reverting to the previous ones will ensure the map is displayed and the page is responsive.

**Comment by aliraza944** [comment]
I have adjusted the height of the Mapbox div. I couldn't get your Mapbox to work but I have fixed the height issue and its responsiveness.<br/ >
I have attached two images with the Mapbox container background to yellow so you can check and verify.
mobile
desktop


**Please let me know if you require anything else**

**Comment by d-rita** [comment]
@aliraza944 thank you for your quick response to this. This is the [current production site](https://galaxy.hotosm.org/) and this is the home page design we have. 

Design:
home page

Retaining the previous width and height specs while maintaining the other proposed changes gave a responsive home page.
home page tested

**Comment by aliraza944** [comment]
I thought that MapBox div is at the bottom.

**Comment by d-rita** [comment]
We have a map background onto which the text and the Explore button appear. Following the[ Mapbox tutorial](https://docs.mapbox.com/help/tutorials/use-mapbox-gl-js-with-react/) here, we render the Mapbox map using the div and add another div with the other components in order to build out the Home page.

**Comment by aliraza944** [comment]
You can test now I have set the banner as absolute and now it appears in front of the map and the map takes all the space.<br />
Here are the images attached for reference.
desktop
 
mobile

**Review by d-rita — APPROVED** [review]
Home, About and Explore Report pages are responsive across various screen sizes.

**Review by itskshitiz321 — APPROVED** [review]

**Comment by petya-kangalova** [comment]
@aliraza944  I wanted to say huge thanks for contributing to the Galaxy UI repo. We are very excited to see new contributors to our repo and really appreciate the input!

**Pull request merged**